testing-utils/README.md

```diff
@@ -1,7 +1,30 @@
 # testing-utils
 
 This library contains testing utilities, such as helper functions or fixtures.
+Any reusable mocks should live here.
 
-## Running unit tests
+## Main utilities
 
-Run `npx nx run testing-utils:unit-test` to execute the unit tests via [Vitest](https://vitest.dev).
+### Mock data
+
+Hardcoded mocks, often referring to a smaller configuration object, live in `utils`.
```

Comment:
It's nice to have links to the folders, makes it easier to find the right one (in this case I thought you meant `packages/utils` at first :man_facepalming:).

```suggestion
Hardcoded mocks, often referring to a smaller configuration object, live in [`utils`](./src/lib/utils).
```

(Same goes for other folder and file references in this README.)

Reply:
I'm dumb, I forgot I can link internal references 🤦 will do.